backend/apps/owasp/migrations/0026_post.py

```
@@ -0,0 +1,30 @@
+# Generated by Django 5.1.7 on 2025-03-12 06:05
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('owasp', '0025_remove_event_country_remove_event_postal_code_and_more'),
```

Ruff (0.8.2)

Single quotes found but double quotes preferred

Replace single quotes with double quotes

(Q000)